### PR TITLE
Defining Event Class and implementing core methods + unittest and verbosity script 

### PR DESCRIPTION
**Pull request by HarryHuCodes**
# Description 
Pull request for tasks from our first sprint and parts of week 2 deliverables

## Deliverable 1
1. Created event class with relevant data members and methods 
2. Applied encapsulation to some attributes and added accessor/mutator methods for them
3. Simulated method overloading using python's substitute ~*arg syntax~ by providing noneType to constructor parameters
4. Made 'event_sampleObj' showcasing a sample event object instantiation, its methods, and usage

## Deliverable 2 
1. Wrote unittest for verifying the functionality of event class and associated methods
2. Created script for running -v verbosity on event_unittest 

## Screenshots

sampleObj event
event_unittest

### CODE REVIEW

File: __pycache__/event_class.cpython-310.pyc
Comment (lucasnchoi) [inline __pycache__/event_class.cpython-310.pyc:1]: As I mentioned in the meeting, let's keep pycaches and other local environment files out of the repo. 
I added a gitignore file in #1 so you'll only have to remove it manually for this PR.

File: __pycache__/event_unittest.cpython-310.pyc
Comment (lucasnchoi) [inline __pycache__/event_unittest.cpython-310.pyc:1]: same as above.

File: event_class.py
Comment (lucasnchoi) [inline event_class.py:1]: Looks good!
The only thing I think might need to be added is some sort of organization ID field (if an event is created by/for an org. But I'll let you and Matt decide how you want to approach this.

File: event_unittest.py
Comment (lucasnchoi) [inline event_unittest.py:1]: Unit tests look good!

**Review by lucasnchoi — CHANGES_REQUESTED** [review]
In general, looks good to me.  Should be good to merge once those pycaches are removed. My only other comment, I think it'd be nice to have all the files related to the event class in a separate folder to help with repo organization but if you want to wait until we have a better of how the models and db are going to work that's also fair.

File: __pycache__/event_class.cpython-310.pyc
Comment (HarryHuCodes) [reply __pycache__/event_class.cpython-310.pyc:1]: Thanks, pycaches have been removed.

File: event_class.py
Comment (HarryHuCodes) [reply event_class.py:1]: Will update once I confirm with Matt and go over the organization class.

**Pull request merged**